app/models/user.rb

```diff
@@ -134,7 +135,7 @@ def edipi
   end
 
   def sec_id
-    va_profile&.sec_id
+    identity.sec_id || va_profile&.sec_id
```

Comment:
Are `identity.sec_id` and `va_profile&.sec_id` always guaranteed to be the same? Looks like we're prioritizing `identity.sec_id` in the event that both are non-nil. Making sure that's intentional / expected.

Reply:
To my knowledge yes, as they both come from the MVI database.  `identity.sec_id` will only be available if the user is authenticated via SSOe, and in that case should be preferred as it will be immediately available and a MVI call will not be needed.  @patrickvinograd can you confirm that my reasoning is correct?

Reply:
identity.sec_id is from the v1 saml response, and va_profile&.sec_id is from MVI.  Both of those are written by the IAM team, so I'd imagine that they're the same.  If someone is using the v0 saml response we wouldn't have the `identity.sec_id` response so we could fall back to MVI

Reply:
Confirming above expectation is correct.